### PR TITLE
proj: Fix urls for website and HEAD

### PR DESCRIPTION
**Pull request by benelsen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Updated the URLs to the website and the repository for HEAD build.
No changes to stable version or resources.

### CODE REVIEW

File: Formula/proj.rb [hunk 1]
```diff
@@ -1,16 +1,23 @@
 class Proj < Formula
-  desc "PROJ.4, a Cartographic Projections Library"
-  homepage "https://trac.osgeo.org/proj/"
+  desc "Cartographic Projections Library"
+  homepage "http://proj4.org"
+
```
Comment (bfontaine) [inline Formula/proj.rb]: Please remove this blank line.

**Review by bfontaine — DISMISSED** [review]

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@benelsen Thanks for your first contribution to Homebrew!

**Pull request merged**